### DISCUSSION TITLE
3.0.0+mc1.18.2

### DISCUSSION THREAD

**Discussion by JustPyrrha** [JustPyrrha/QForward#8] [Announcements]
- Update to 1.18.2
- Switch to Quilt
- Drop legacy forwarding support
- Rebrand to QForward

<hr /><em>This discussion was created from the release <a href='https://github.com/JustPyrrha/QForward/releases/tag/'>3.0.0+mc1.18.2</a>.</em>

**Comment by voruti** [comment]
Why was legacy forwarding support removed? I would like to use this ...
